umbra-js/package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-config-prettier": "^8.2.0",
     "eslint-plugin-chai-friendly": "^0.6.0",
     "eslint-plugin-import": "^2.20.2",
-    "hardhat": "^2.12.4",
+    "hardhat": "2.17.0",
```

Comment:
oh I see, maybe only umbra-js was affected by the bug?

Reply:
Yes, only having issues inside `umbra-js,` see comment above https://github.com/ScopeLift/umbra-protocol/pull/577#discussion_r1284556417